ccloud/docs/beginner-cloud.rst

```diff
@@ -2,35 +2,18 @@
 
 .. _ccloud-cli-tutorial:
 
-Tutorial: Confluent CLI
-=======================
+Tutorial: Use |confluent-cli| with |ccloud|
+===========================================
 
-Overview
---------
-
-This tutorial shows you how to use the `Confluent CLI
-<https://docs.confluent.io/confluent-cli/current/install.html>`__ to interact with
-your `Confluent Cloud <https://www.confluent.io/confluent-cloud/tryfree/>`__ cluster. It uses real
+This tutorial shows you how to use the |confluent-cli| to interact with
+your |ccloud| cluster. It uses real
 resources in |ccloud|, and it creates and deletes topics, service accounts,
-credentials, and ACLs. Following the workflow in this tutorial, you accomplish
-the following steps:
-
--  `Create a new Confluent Cloud environment`_
--  `Create a new Confluent Cloud cluster`_
--  `Create a new API key/secret pair for user`_
--  `Produce and consume records with Confluent CLI`_
--  `Create a new service account with an API key/secret pair`_
--  `Run a Java producer without ACLs`_
--  `Run a Java producer with ACLs`_
--  `Run a Java producer with a prefixed ACL`_
--  `Run a fully managed Confluent Cloud connector`_
--  `Run a Java consumer with a Wildcard ACL`_
--  `Clean up Confluent Cloud resources`_
+credentials, and ACLs.
 
 Prerequisites
 -------------
 
--  `Confluent Cloud <https://www.confluent.io/confluent-cloud/tryfree/>`__
+-  `Confluent Cloud <https://www.confluent.io/get-started/?product=cloud>`__
 
 -  Local `install of Confluent CLI
    <https://docs.confluent.io/confluent-cli/current/install.html>`__ (v3.0.0 or later)
```

Comment:
```suggestion
   <https://docs.confluent.io/confluent-cli/current/install.html>`__ (version 3.0.0 or later)
```

Reply:
FYI i made this change over in the new location in docs-confluent-cli. https://github.com/confluentinc/docs-confluent-cli/pull/613